sycl/doc/extensions/supported/sycl_ext_intel_usm_address_spaces.asciidoc

```diff
@@ -1,139 +1,206 @@
 = sycl_ext_intel_usm_address_spaces
 
-== Introduction
-This extension introduces two new address spaces and their corresponding multi_ptr specializations.  
-These address spaces are subsets of the global address space and are added to enable users to provide more optimization information to their compiler.  
+:source-highlighter: coderay
+:coderay-linenums-mode: table
 
-IMPORTANT: This specification is a draft.
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+:dpcpp: pass:[DPC++]
 
-NOTE: Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by permission by Khronos.
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
 
-NOTE: This document is better viewed when rendered as html with asciidoctor.  GitHub does not render image icons.
-This document describes an extension to the SYCL USM extension that adds new explicit address spaces for the possible locations that USM pointers can be allocated.  Users can create pointers that point into these address spaces explicitly in order to pass additional information to their compiler so as to enable optimizations.  
 
 == Notice
-Copyright (c) 2020 Intel Corporation.  All rights reserved.
 
-== Status
+[%hardbreaks]
+Copyright (C) 2022 Intel Corporation.  All rights reserved.
+
+Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks
+of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by
+permission by Khronos.
 
-Draft
 
-This is a preview extension specification, intended to provide early access to a feature for review and community feedback. When the feature matures, this specification may be released as a formal extension.
+== Contact
 
-Because the interfaces defined by this specification are not final and are subject to change they are not intended to be used by shipping software products.
+To report problems with this extension, please open a new issue at:
 
-== Version
+https://github.com/intel/llvm/issues
 
-Built On: {docdate} +
-Revision: 2
 
 == Dependencies
 
-This extension is written against the SYCL 2020 specification, Revision 3.
+This extension is written against the SYCL 2020 revision 6 specification.  All
+references below to the "core SYCL specification" or to section numbers in the
+SYCL specification refer to that revision.
+
+If SPIR-V is used by the implementation, this extension also requires support
+for the SPV_INTEL_usm_storage_classes SPIR-V extension.
```

Comment:
This seems like an implementation detail to me, and it should not be part of the SYCL extension.  Our other SYCL extension spec don't list the underlying SPIR-V extensions that they make use of.

Reply:
It has been removed!